tests/functional/upgrade/test_configuration.py

```diff
@@ -108,10 +109,14 @@ def test_max_unavaialable_cephfs(upgrade_stats):
         namespace=config.ENV_DATA["cluster_namespace"],
         resource_name=constants.ROOK_OPERATOR_CONFIGMAP,
     ).get()
-    config_value = configmap.get("data").get(
-        "CSI_CEPHFS_PLUGIN_UPDATE_STRATEGY_MAX_UNAVAILABLE"
-    )
-    assert config_value == upgrade_stats["odf_upgrade"]["cephfs_max_unavailable"]
+    if (
+        config.UPGRADE.get("CSI_CEPHFS_PLUGIN_UPDATE_STRATEGY_MAX_UNAVAILABLE")
```

Comment:
Same as above, the `CSI_CEPHFS_PLUGIN_UPDATE_STRATEGY_MAX_UNAVAILABLE` should be lower case here.

Reply:
Incorporated review comments.